models/Element/Tag.php

```diff
@@ -350,7 +359,14 @@ public function delete(): void
     {
         $this->dispatchEvent(new TagEvent($this), TagEvents::PRE_DELETE);
 
-        $this->getDao()->delete();
+        $deletedTagIds = $this->getDao()->delete();
```

Comment:
we could potentially then pass this `$deletedTagIds` array to `TagEvents::POST_DELETE` but it might be for an improvement